src/superphot_plus/data_generation/make_fake_spp_data.py

```diff
@@ -117,6 +117,31 @@ def create_prior(cube, tdata):
     return cube
 
 
+def ztf_noise_model(mag, band, snr_range_g = [1,10], snr_range_r=[1,10]):
+    """
+    A very, very simple noise model which assumes the dimmest magnitude is at SNR = 1, and the brightest mad is at SNR = 10
+
+    inputs: 
```

Comment:
Small formatting request: For consistency we use the same formatting we have been using in other doc strings?

```
    Parameters
    ----------
    parameter_name : type
        Description
```